README.md

```diff
@@ -62,7 +62,7 @@ To build the Riak Java Client, you must have the correct version of the riak_pb
 
 ```
 git clone https://github.com/basho/riak_pb
-git checkout java-2.1.1.0
+git checkout 2.1.2.0
```

Comment:
@srgg I have a branch that does that - There were some changes to the TSQuery command since we switched to term-to-binary support.  https://github.com/basho/riak-java-client/tree/merge-dpclient-ts1.3

We need a test around using a coverage context with a TS query request.  I need to see if I wrote this line correctly:  https://github.com/basho/riak-java-client/blob/bd6cf991a3bae2808eeb05b6b4a7951e9e089cff/src/main/java/com/basho/riak/client/core/codec/TermToBinaryCodec.java#L82-L82